src/scrub.S

```diff
@@ -26,10 +68,7 @@ _metal_memory_scrub:
 #endif
     ret
 
-/*
- * Initialize all memories available to zero
- * This must be call before seting up any stack(s)
- */
+/* Initialize memories to zero */
```

Comment:
The comment about doing this before setting up the stack is still true, so it should be left in place.

Reply:
Fixed.